datalake/src/main/java/com/sequenceiq/datalake/service/sdx/ProvisionerService.java

```diff
@@ -118,9 +118,10 @@ public void waitCloudbreakClusterDeletion(Long id, PollingConfig pollingConfig)
                                 if (Status.DELETE_FAILED.equals(cluster.getStatus())) {
                                     // it's a hack, until we implement a nice non-async terminate which can return with flowid..
                                     // if it is implemented, please remove this
-                                    if (deleteFailedCount.getAndIncrement() > DELETE_FAILED_RETRY_COUNT) {
+                                    if (deleteFailedCount.getAndIncrement() >= DELETE_FAILED_RETRY_COUNT) {
                                         return AttemptResults.breakFor(
-                                                "Cluster deletion failed '" + sdxCluster.getClusterName() + "', " + stackV4Response.getStatusReason()
+                                                "Cluster deletion failed '" + sdxCluster.getClusterName() + "', "
```

Comment:
I think overall would be great to use the platform terms like : Data Lake deletion failed